custom_components/tuya_local/devices/makegood_double_power_point_with_energy.yaml

```diff
@@ -0,0 +1,84 @@
+name: Makegood double power point
```

Comment:
If they are sold under different brands, remove the brand from here.   If you can find the product ids for the various brands you have access to, then listing them here under a `products` section would help with future plans for discovery.